src/patternfly/components/Switch/switch.scss

```diff
@@ -19,6 +19,8 @@
   --pf-c-switch__toggle--before--BorderRadius: var(--pf-c-switch__toggle--before--Width);
   --pf-c-switch__toggle--before--BoxShadow: var(--pf-global--BoxShadow--md);
   --pf-c-switch__toggle--before--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-switch__toggle--before--Top: calc(#{var(--pf-c-switch--Height)} / 2 * (1 - 1 / #{var(--pf-global--golden-ratio)}));
+  --pf-c-switch__toggle--before--Left: var(--pf-c-switch__toggle--before--Top);
   --pf-c-switch__label--Color: var(--pf-global--Color--dark-100);
```

Comment:
The variables for at least `--pf-c-switch__toggle` and `--pf-c-switch__label` could be reordered to match the order in which they're used down below, but that's not really important (or this PR's job).

Reply:
Ooh fancy! I don't think we do that as a thing (or I haven't anyways), but sure why not?